packages/api-server/api_server/routes/internal.py

```diff
@@ -15,6 +15,9 @@
 
 
 async def process_msg(msg: Dict[str, Any], fleet_repo: FleetRepository) -> None:
+    if "type" not in msg:
+        logger.debug("Ignoring message, 'type' must include in msg field")
+        return
     payload_type: str = msg["type"]
     if not isinstance(payload_type, str):
         logger.error("error processing message, 'type' must be a string")
```

Comment:
Can you add `logger.warn(msg)` to this and the above `type` check? Having it print out the msg will be very helpful.